tests/unit/s2n_tls13_hybrid_shared_secret_test.c

```diff
@@ -408,7 +408,8 @@ int main(int argc, char **argv) {
              * the client & server traffic secrets */
             DEFER_CLEANUP(struct s2n_tls13_keys secrets = {0}, s2n_tls13_keys_free);
             EXPECT_SUCCESS(s2n_tls13_keys_init(&secrets, test_vector->cipher_suite->prf_alg));
-            EXPECT_SUCCESS(s2n_tls13_derive_early_secrets(&secrets));
+            struct s2n_psk *psk = NULL;
+            EXPECT_SUCCESS(s2n_tls13_derive_early_secrets(&secrets, psk));
```

Comment:
Might be cleaner if you just do `EXPECT_SUCCESS(s2n_tls13_derive_early_secrets(&secrets, NULL));`